### ISSUE TITLE
feature: Support mouse XButtons 1 & 2

### ISSUE THREAD

**Issue by AlexVild** [AscensionGameDev/Intersect-Engine#1865]
### Description

The mice XButtons, commonly used these days as browser back/forward on gaming mice, are not fully supported in intersect and cannot be bound for hotkeys or used for custom application.

This is because our mice event handlers do not listen for these buttons.

### Intended Use-Case

It would be nice to have these buttons, as a player, for extended keybindings, especially for hotkeys. It could also be nice as a developer if you write some custom UI and want back/forward support with those buttons.

### Duplicate Check

- [X] This feature request is not a duplicate to the best of my knowledge.
